tests/cache_tests.py

```diff
@@ -1,7 +1,9 @@
-from unittest import TestCase
+from unittest import TestCase, skipIf
 from . import utils
 import memcache
+import os
```

Comment:
So you want it to become the following or something else?
```python
import os
from unittest import TestCase, skipIf
import memcache
from . import utils
```
Is the styleguide documented somewhere?